tests/_support/AcceptanceTester.php

```diff
@@ -141,4 +143,10 @@ public function searchFor($query, $delay = 0, $element = '#search_input', $butto
     $I->click($button);
   }
 
+  public function switchToNextTab($offset = 1) {
+    $this->_switchToNextTab($offset);
+
+    // workaround for frozen tabs when opened by clicking on links
+    $this->wait(1);
```

Comment:
I don't understand how could that help.  But the build is green, so I am happy.

Reply:
Me neither. I tried to wait for some text on the page but that did not help. It seems that if Selenium touches the tab too early it can get frozen (or something like that) - I see just a white tab with the correct URL but even with manual page refresh it stays frozen. Googled a lot but didn't find any issues. I may create one 🤔